packages/react-components/src/components/styles/ChatMessageComponent.styles.ts

```diff
@@ -3,27 +3,6 @@
 
 import { mergeStyles, IStyle, FontWeights } from '@fluentui/react';
 
-export const chatMessageStyle = mergeStyles({
```

Comment:
Good, this is just a copy&paste part from legacy code when I wrote this, don't know what does this for, good to see we remove it